plugins/content/bower/index.js

```diff
@@ -1145,10 +1149,10 @@ function handleUploadedPlugin (req, res, next) {
 
                   function sendResponse() {
                     res.statusCode = 200;
-                    return res.json({ 
-                      success: true, 
-                      pluginType: pluginType, 
-                      message: 'successfully added new plugin' 
+                    return res.json({
+                      success: true,
+                      pluginType: pluginType,
+                      message: 'successfully added new plugin'
```

Comment:
Is it worth putting this into the translation files as well?

Reply:
This isn't actually displayed anywhere/used so I will remove it